src/main/java/com/example/cult_of_tim/cultoftim/service/impl/PromotionMockService.java

```diff
@@ -37,17 +38,29 @@ public List<Promotion> getUserPromotions(Long userID) {
 
     @Override
     public boolean isGlobal(Long promotionID) {
-        return promotionRepository.existsByIdAndGlobal(promotionID, true);
+        return promotionRepository.existsByIdAndGlobalPromotion(promotionID, true);
```

Comment:
First, change globalPromotion to global (boolean in Promotion).
Second, make method better in PromotionRepository, read comment there.
Third, insert here. If for some reason you need existsbysomething, don't put it in isGlobalMethod.